docs/components/concepts/signals.md

```diff
@@ -13,8 +13,9 @@ respond accordingly.
 
 You can broadcast signals in two ways:
```

Comment:
```suggestion
You can broadcast signals in three ways:
```

And same for 8.6. (Or we just change it to "Several" and never worry about the count again!)

Reply:
Yes, great idea! Counting is lame. 😝